src/services/api/messages.rs

```diff
@@ -43,3 +43,56 @@ impl StarknetMessageToL1 {
         finalized_hash.as_slice().to_vec()
     }
 }
+
+#[test]
+fn create_starknet_message_to_l1() {
+    let from_address: Address = Address(42.into());
+    let to_address: Address = Address(1729.into());
+    let payload: Vec<Felt> = Vec::from([Felt::new(1), Felt::new(2), Felt::new(3), Felt::new(4)]);
```

Comment:
```suggestion
    let payload = vec!([Felt::new(1), Felt::new(2), Felt::new(3), Felt::new(4)]);
```
You can use the macro vec! for these cases.
Also is the type annotation necessary here?

Reply:
i would go further and also replace `Felt::from()` with `into()` thus it will be more clear. Type annotation is not needed as long as the compiler is able to infer the type of `into()` or of the vector.